1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,83 @@
-// Write your code here
+//array containing imgs
+let images = [
+  "./reel-images/boat.jpg",
+  "./reel-images/camel.jpg",
+  "./reel-images/blossoms.jpg",
+  "./reel-images/roadtravel.jpg",
+  "./reel-images/japan.jpg",
+  "./reel-images/guatemala.jpg",
+  "./reel-images/germany.jpg"
+];
+
+//navigate through imgs/display different imgs
+let arrayIndex = 0;
+
+//hold reference to html doc img tag
+let heroImage;
+
+function manualSlideshow() {
+  heroImage = document.getElementById("main-image");
+  //obtain button classes
+  let prevBtn = document.getElementById("prev");
+  let nextBtn = document.getElementById("next");
+
+  /*checks to see if the button who’s ID is 
+  buttonPrevious supports the addEventListener method*/
+  if (prevBtn.addEventListener) {
```

Comment:
why did you add this check around whether to use `addEventListener` vs `attachEvent`?